lms/djangoapps/instructor/tasks.py

```diff
@@ -20,6 +21,7 @@
 
 
 @shared_task(base=LoggedTask, ignore_result=True)
+@set_code_owner_attribute
```

Comment:
@jinder1s: Warning, there have been a handful of tasks where this decorator broke it, based on how the task was coded. So, you or @Dillon-Dumesnil may be on the lookout to ensure this keeps working as intended.